service/vochain.go

```diff
@@ -146,6 +150,30 @@ func Vochain(vconfig *config.VochainCfg, waitForSync bool,
 		)
 	}
 
+	// get the zk Circuits VerificationKey files
+	vnode.ZkVKs = []*snarkTypes.Vk{}
+	for i, cc := range vochain.Genesis[vconfig.Chain].CircuitsConfig {
+		log.Infof("downloading zk-circuits-artifacts index: %d", i)
+
+		// download VKs from CircuitsConfig
+		ctx, cancel := context.WithTimeout(context.Background(), time.Duration(time.Minute*1))
+		defer cancel()
+		err = zkartifacts.DownloadVKFile(ctx, cc)
+		if err != nil {
+			return
+		}
+
+		// parse VK and store it into vnode.ZkVKs
+		log.Infof("parse VK from file into memory. CircuitArtifact index: %d", i)
+		var vk *snarkTypes.Vk
+		vk, err = zk.LoadVkFromFile(filepath.Join(cc.LocalDir, cc.CircuitPath, zkartifacts.FilenameVK))
```

Comment:
use `:=` and avoid the `var`?

Reply:
I used the `var` because the `err` is the one returned in the function, and if I use `vk, err :=` the compiler complains that 'result parameter err not in scope at return`. But I'm down to use a better way instead than the current one

Reply:
oh, right, named result parameters and naked returns. I'd strongly recommend avoiding those :) See https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters and https://github.com/golang/go/wiki/CodeReviewComments#naked-returns.

If I see a naked return 50 or 100 lines after the func declaration, I have no idea what I'm returning.

Reply:
Agree with what you mention, in this case this *naked return* was there prior to this PR, but I'm very agree with what you mentioned, so I'll update the function to not use the naked return.